core/src/ldml/ldml_processor.cpp

```diff
@@ -246,6 +214,7 @@ ldml_processor::process_event(
           }
         }
         if (last_char == 0UL) {
+          // couldn't find the last char
```

Comment:
```suggestion
          /*
            We couldn't find a character at end of context (context is empty),
            so we'll pass the backspace keystroke on to the app to process; the
            app might want to use backspace to move between contexts or delete 
            a text box, etc. Or it might be a legacy app and we've had our caret
            dumped in somewhere unknown, so we will have to depend on the app to
            be sensible about backspacing because we know nothing.
          */
```